### PR TITLE
SA-2865 - Update CrowdStrike Mac Command to include license agreement

### PR DESCRIPTION
**Pull request by gweinjc**
## Issues
* [SA-2865](https://jumpcloud.atlassian.net/browse/SA-2865) - Update CrowdStrike Mac Command to include license agreement

## What does this solve?
When installing the CrowdStrike Sensor via Commands, the installation was completing as expected, however the sensor was never checking into the CrowdStrike console due to the license command not being included. See CrowdStrike's installation documentation for more information (https://www.crowdstrike.com/blog/tech-center/how-to-install-the-falcon-sensor-for-mac/)

## Is there anything particularly tricky?
N/A

## How should this be tested?
Run the command and supply your CrowdStrike URL as well as the CID
Validate that the device is checking into the CrowdStrike console

## Screenshots
image

### CODE REVIEW

File: PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install CrowdStrike Falcon Agent.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ mac
 
 ```bash
 #!/bin/bash
-DownloadUrl="https://path/to/url.pkg"
+DownloadUrl=""
```
Comment (jworkmanjc) [inline PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install CrowdStrike Falcon Agent.md:13]:
```suggestion
DownloadUrl="https://path/to/url.pkg"
```

Comment (jworkmanjc) [reply PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install CrowdStrike Falcon Agent.md:13]: Nitpicky but I like having a placeholder URL in there

**Review by jworkmanjc — APPROVED** [review]
Thanks for the conversation yesterday. I think the only suggestion I have is to add back the placeholder URL. Else I think this is a good flow and will force admins to follow the suggested workflow of adding the policy first, then install the software!

**Review by kmaranionjc — APPROVED** [review]
Test works. Nice work!

**Pull request merged**